xray/commands/offlineupdate/offlineupdate.go

```diff
@@ -378,14 +394,46 @@ func getFilesList(updatesUrl string, flags *OfflineUpdatesFlags) (vulnerabilitie
 	return
 }
 
+// ValidStreams represents the valid values that can be provided to the 'stream' flag during offline updates.
+type ValidStreams struct {
+	StreamsMap map[string]bool
+}
+
+func NewValidStreams() *ValidStreams {
+	validStreams := &ValidStreams{StreamsMap: map[string]bool{}}
+	validStreams.StreamsMap[validStreams.GetPublicDataStream()] = true
+	validStreams.StreamsMap[validStreams.GetExposuresStream()] = true
+	validStreams.StreamsMap[validStreams.GetContextualAnalysisStream()] = true
+	return validStreams
+}
+
+func (vs *ValidStreams) GetValidStreamsList() string {
```

Comment:
GetValidStreamsList --> GetValidStreamsString

Also, let's add a unit test for this function.